src/features/import/utils/prepareImportOperations.ts

```diff
@@ -63,12 +65,9 @@ export default function prepareImportOperations(
         if (column.kind === ColumnKind.ORGANIZATION) {
           column.mapping.forEach((mappedColumn) => {
             if (mappedColumn.value === row.data[colIdx]) {
-              if (!personImportOps[rowIndex].organizations) {
-                personImportOps[rowIndex].organizations = [];
-              }
-              personImportOps[rowIndex].organizations = personImportOps[
-                rowIndex
-              ].organizations?.concat(mappedColumn.orgId as number);
+              personImportOps[rowIndex].organization = [
```

Comment:
This should not have been renamed. The interface of these operators is defined by the backend, and we cannot just change it on the frontend. It needs to be `organizations` for this code to work with the backend.